src/documentation/pages/components/forms/range-slider/examples/_multiple_consistent.vue

```diff
@@ -0,0 +1,69 @@
+<template lang="pug">
+  <ComponentExample title="Multiple Labels" id="multiple-labels" :tabs="exampleTabs">
+    .chi-form__item(slot="example")
+      label(class="chi-label", id="range-label", for="range06") Range label
+      .chi-input__wrapper.-d--flex
+        span.-text.-mr--2 200GB
+        .chi-input__wrapper.-w--100
+          input(type="range", class="chi-range-slider", ref="range")
```

Comment:
On label you are using `for="range06"`
```suggestion
          input(type="range", class="chi-range-slider", ref="range" id="range06")
```